lib/raix/configuration.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module Raix
+  # The Configuration class holds the configuration options for the Raix gem.
+  class Configuration
+    # The temperature option determines the randomness of the generated text.
+    # Higher values result in more random output.
+    def temperature
+      get_with_fallback(:temperature)
+    end
+    attr_writer :temperature, :max_tokens, :max_completion_tokens, :model, :openrouter_client, :openai_client
```

Comment:
My editor messed this up. 🤔 

I do wonder if it would make sense to at least sprinkle in some meta programming here so that you could write `attr_accessor_with_fallback :temperature` and such?